package.json

```diff
@@ -39,6 +40,7 @@
   ],
   "scripts": {
     "test": "jest",
+    "test-watch": "jest --watch",
```

Comment:
`npm test -- --watch` does the same thing, and isn't that much longer than `npm run test-watch`.

Reply:
Neat trick, thanks. Removed this script.